// GENERATED CODE - DO NOT EDIT
// Generated by schema2code proto generator

syntax = "proto3";

package proto;

option go_package = "maistro/proto";

import "chat_message.proto";

// ChatResponse represents a response from the Ollama chat API
message ChatResponse {
  bool done = 1;
  ChatMessage message = 2;
  string created_at = 3;
  string model = 4;
  repeated double context = 5;
  string done_reason = 6;
  double total_duration = 7;
  double load_duration = 8;
  double prompt_eval_count = 9;
  double prompt_eval_duration = 10;
  double eval_count = 11;
  double eval_duration = 12;
}
